src/mangosd/mangosd.conf.dist.in

```diff
@@ -37,13 +37,13 @@ ConfVersion=2015090801
 #                    hostname;port;username;password;database
 #                    .;/path/to/unix_socket/DIRECTORY or . for default path;username;password;database - use Unix sockets at Unix/Linux
 #
-#	LoginDatabaseConnections
-#	WorldDatabaseConnections
-#	CharacterDatabaseConnections
-#		 Amount of connections to database which will be used for SELECT queries. Maximum 16 connections per database.
-#		 Please, note, for data consistency only one connection for each database is used for transactions and async SELECTs.
-#		 So formula to find out how many connections will be established: X = �_connections + 1
-#		 Default: 1 connection for SELECT statements
+#    LoginDatabaseConnections
+#    WorldDatabaseConnections
+#    CharacterDatabaseConnections
+#         Amount of connections to database which will be used for SELECT queries. Maximum 16 connections per database.
+#         Please, note, for data consistency only one connection for each database is used for transactions and async SELECTs.
+#         So formula to find out how many connections will be established: X = �_connections + 1
```

Comment:
Yeah, I just get a '?' - I doubt that's the correct symbol. Maybe someone else should remove those tabs then. But they're there! =)